test/transceivercontroller/SimulcastTransceiverController.test.ts

```diff
@@ -71,6 +71,27 @@ describe('SimulcastTransceiverController', () => {
       expect(peer.getTransceivers().length).to.equal(0);
     });
 
+    it('can set up transceivers without the meeting session context', () => {
+      tc = new SimulcastTransceiverController(logger, context.browserBehavior);
+      const peer: RTCPeerConnection = new RTCPeerConnection();
+      tc.setPeer(peer);
+      tc.setupLocalTransceivers();
+      expect(peer.getTransceivers().length).to.equal(2);
+
+      const transceivers = peer.getTransceivers();
+      expect(transceivers.length).to.equal(2);
+
+      const audioTransceiver = transceivers[0];
+      expect(audioTransceiver.direction).to.equal('inactive');
+      expect(audioTransceiver.receiver.track.kind).to.equal('audio');
+      expect(audioTransceiver.sender.track.kind).to.equal('audio');
+
+      const videoTransceiver = transceivers[1];
+      expect(videoTransceiver.direction).to.equal('inactive');
+      expect(videoTransceiver.receiver.track.kind).to.equal('video');
+      expect(videoTransceiver.sender.track.kind).to.equal('video');
+    });
```

Comment:
nit: maybe also check that `tc['audioRedWorker']` is null/undefined?

Reply:
It looks like I can't do that since `audioRedWorker` would need to be `protected` instead of `private` in `DefaultTransceiverController` for it to be accessible (even with that syntax) in `SimulcastTransceiverController`